src/main/java/nl/nn/testtool/web/api/TestToolApi.java

```diff
@@ -50,227 +50,214 @@
 
 @Path("/" + ApiServlet.LADYBUG_API_PATH + "/testtool")
 public class TestToolApi extends ApiBase {
-	private @Setter @Inject @Autowired TestTool testTool;
-	private @Setter @Inject @Autowired MetadataExtractor metadataExtractor;
-	private @Setter @Inject @Autowired ReportXmlTransformer reportXmlTransformer;
-	private @Setter @Inject @Autowired Views views;
-	private String defaultTransformation;
-
-	@PostConstruct
-	public void init() {
-		defaultTransformation = reportXmlTransformer.getXslt();
-	}
-
-	public String getDefaultTransformation() {
-		return defaultTransformation;
-	}
-
-	/**
-	 * @return Response containing test tool data.
-	 */
-	@GET
-	@Path("/")
-	@Produces(MediaType.APPLICATION_JSON)
-	public Response getInfo() {
-		Map<String, Object> info = getTestToolInfo();
-		return Response.ok(info).build();
-	}
-
-	@GET
-	@Path("/reset")
-	@Produces(MediaType.APPLICATION_JSON)
-	public Response resetInfo() {
-		testTool.reset();
-		Map<String, Object> info = getTestToolInfo();
-		return Response.ok(info).build();
-	}
-
-	public Map<String, Object> getTestToolInfo() {
-		Map<String, Object> map = new HashMap<>(4);
-		map.put("generatorEnabled", testTool.isReportGeneratorEnabled());
-		map.put("estMemory", testTool.getReportsInProgressEstimatedMemoryUsage());
-		map.put("regexFilter", testTool.getRegexFilter());
-		map.put("reportsInProgress", testTool.getNumberOfReportsInProgress());
-		map.put("stubStrategies", testTool.getStubStrategies());
-		return map;
-	}
-
-	/**
-	 * Change settings of the testtool.
-	 *
-	 * @param map New settings that can contain (generatorEnabled, regexFilter)
-	 * @return The response after changing the settings.
-	 */
-	@POST
-	@Path("/")
-	@Consumes(MediaType.APPLICATION_JSON)
-	public Response updateInfo(Map<String, String> map) {
-		if (map.isEmpty()) {
-			return Response.status(Response.Status.BAD_REQUEST).entity("No settings have been provided - detailed error message - The settings that have been provided are " + map).build();
-		}
+    private @Setter
+    @Inject
```

Comment:
What happened on this line (newlines added)?

Reply:
It's the default java formatting again

Reply:
Again, please change :)